apps/events/tests/utils.py

```diff
@@ -1,17 +1,55 @@
 from django.contrib.auth.models import Group
+from django.contrib.contenttypes.models import ContentType
 from django_dynamic_fixture import G
 from guardian.shortcuts import assign_perm, get_perms_for_model
 
-from ..models import TYPE_CHOICES, AttendanceEvent, Event
+from apps.payment.models import Payment, PaymentDelay, PaymentRelation
+
+from ..models import TYPE_CHOICES, AttendanceEvent, Attendee, Event
 from ..utils import get_organizer_by_event_type
 
 
 def generate_event(event_type=TYPE_CHOICES[1][0]):
-    event = G(Event, event_type=event_type, organizer=get_organizer_by_event_type(event_type))
+    event = G(Event, event_type=event_type,
+              organizer=get_organizer_by_event_type(event_type))
     G(AttendanceEvent, event=event)
     return event
 
 
+def generate_payment(event):
+    return G(
+        Payment,
+        object_id=event.id,
+        content_type=ContentType.objects.get_for_model(AttendanceEvent)
+    )
```

Comment:
Some crazy formatting here, but whatever floats your ⛵️

Reply:
To be honest I like it better than what autopep8 produces.
`object_id=event.id` could've been put on the line above, but I think it's clearer when each kwarg is on its own line. 

I especially don't like how arguments should line up with the line above.  http://docs.python-requests.org/en/latest/dev/contributing/#kenneth-reitz-s-code-style

Anyway doesn't really matter and I don't think we have any consist styling when it comes to this anyway.

Reply:
Nothing to keep this from being merged at all, no.

We could look into using [yapf](https://github.com/google/yapf) if we want to enforce some rules regarding this.